lib/salus/scanners/base.rb

```diff
@@ -178,6 +178,16 @@ def validate_bool_option(keyword, value)
       false
     end
 
+    def validate_int_option(keyword, value)
+      return true if value&.to_s&.match?(/\d+/)
+
+      report_warn(:scanner_misconfiguration, "Expecting #{keyword} to be a numeric "\
```

Comment:
Can you update `numeric` to `int`?  because `numeric` includes decimals like `1.234`, but we only want ints.
In fact, maybe `non-negative int`?